gitless/cli/gl_init.py

```diff
@@ -25,14 +25,23 @@ def parser(subparsers, _):
       help=(
           'an optional remote repo address from where to read to create the '
           'local repo'))
+  init_parser.add_argument(
+      '-o', '--only', nargs='*',
```

Comment:
shouldn’t this be + instead of *? That way users will get an error if they do gl init -o but forget to pass any branches as input